podcast.rss

```diff
@@ -7012,7 +7012,7 @@
     <item>
       <title>596  സൂറഃ നാസിയാത്ത്  ആയത്ത്  6-9 part 1_a</title>
```

Comment:
Let's maintain consistency between words. For e.g., below title had more than single space between words.

`<title>597   സൂറഃ നാസിയാത്ത്   ആയത്ത് 5-9 Part 2_a</title>`